### ISSUE TITLE
The inference result is empty of TensorRT 8.2.4.2 when running ubuntu20.04 on GPU 3060

### ISSUE THREAD

**Issue by wafaer** [NVIDIA/TensorRT#3506]
## Environment

**TensorRT Version**:8.2.4.2

**NVIDIA GPU**:3060

**NVIDIA Driver Version**:525.147

**CUDA Version**:11.4

**CUDNN Version**:8.2.4

Operating System:ubuntu20.04

Python Version (if applicable):

Tensorflow Version (if applicable):

PyTorch Version (if applicable):

Baremetal or Container (if so, version):

## Relevant Files

**Model link**:[Modify basic code ](https://github.com/wang-xinyu/tensorrtx/blob/master/yolov5/src/model.cpp)

The code I added：

ILayer* hSwish(INetworkDefinition *network, ITensor& input) {
    Weights emptywts{ DataType::kFLOAT, nullptr, 0 };
    float* scval = reinterpret_cast<float*>(malloc(sizeof(float)));
    scval[0] = 1.0 / 6.0;
    Weights scale{ DataType::kFLOAT, scval, 1 };
    float* shval = reinterpret_cast<float*>(malloc(sizeof(float)));
    shval[0] = 0.5;
    Weights shift{ DataType::kFLOAT, shval, 1 };
    float* pval = reinterpret_cast<float*>(malloc(sizeof(float)));
    pval[0] = 1.0;
    Weights power{ DataType::kFLOAT, pval, 1 };
    auto clip = network->addActivation(input, ActivationType::kCLIP);
    clip->setAlpha(-3.0);
    clip->setBeta(3.0);
    auto sc = network->addScale(*clip->getOutput(0), ScaleMode::kUNIFORM, shift, scale, emptywts);
    auto hs = network->addElementWise(*sc->getOutput(0), input, ElementWiseOperation::kPROD);
    return hs;
}

static ILayer* convBlock2(INetworkDefinition *network, std::map<std::string, Weights>& weightMap, ITensor& input,int outch, int ksize, int s,
                          int g, std::string lname1,std::string lname2)
{
    Weights emptywts{ DataType::kFLOAT, nullptr, 0 };
    int p = 0;

    IConvolutionLayer* conv1 = network->addConvolutionNd(input, outch, DimsHW{ ksize, ksize }, weightMap[lname1 + ".weight"], emptywts);
    assert(conv1);
    conv1->setStrideNd(DimsHW{ s, s });
    conv1->setPaddingNd(DimsHW{ p, p });
    conv1->setNbGroups(g);
    conv1->setName((lname1).c_str());

    IScaleLayer* bn1 = addBatchNorm2d(network, weightMap, *conv1->getOutput(0), lname2 + ".bn1", 1e-3);

    auto hsw = hSwish(network, *bn1->getOutput(0));
    assert(hsw);
    return hsw;
}

static ILayer* convBlock3(INetworkDefinition *network, std::map<std::string, Weights>& weightMap, ITensor& input, int outch, int ksize, int s, int g, std::string lname)
{
    Weights emptywts{ DataType::kFLOAT, nullptr, 0 };
    int p = 0;
    IConvolutionLayer* conv1 = network->addConvolutionNd(input, outch, DimsHW{ ksize, ksize }, weightMap[lname + ".weight"], emptywts);
    assert(conv1);
    conv1->setStrideNd(DimsHW{ s, s });
    conv1->setPaddingNd(DimsHW{ p, p });
    conv1->setNbGroups(g);
    conv1->setName((lname).c_str());

    // silu = x * sigmoid
    auto sig = network->addActivation(*conv1->getOutput(0), ActivationType::kSIGMOID);
    assert(sig);

    auto ew = network->addElementWise(*conv1->getOutput(0), *sig->getOutput(0), ElementWiseOperation::kPROD);

    return ew;
}

static ILayer* CoordAtt(INetworkDefinition *network, std::map<std::string, Weights>& weightMap, ITensor& input, int& inp, std::string lname)
{
    Dims dims = input.getDimensions();
    int h_size = dims.d[1];
    int w_size = dims.d[2];

    /// pool_h *************************************************************************************************************
    auto h_pool = network->addPoolingNd(input, PoolingType::kAVERAGE,DimsHW{1,h_size});
    h_pool->setStrideNd(DimsHW{1,1});
    h_pool->setPaddingNd(DimsHW{0,0});
    ITensor* h_value = h_pool->getOutput(0);

    /// pool_w *************************************************************************************************************
    auto w_pool = network->addPoolingNd(input, PoolingType::kAVERAGE,DimsHW{w_size,1});
    w_pool->setStrideNd(DimsHW{1,1});
    w_pool->setPaddingNd(DimsHW{0,0});
    ITensor* w_value = w_pool->getOutput(0);

    Permutation permutation1 = {0,2,1};
    IShuffleLayer* shuffleLayer1 = network->addShuffle(*w_value);
    shuffleLayer1->setFirstTranspose(Permutation(permutation1));
    ITensor* perm1 = shuffleLayer1->getOutput(0);

    /// torch.cat *************************************************************************************************************
    ITensor* sum_input_tensors[] = {h_value, perm1};
    IConcatenationLayer* concatLayer = network->addConcatenation(sum_input_tensors,2);
    concatLayer->setAxis(1);
    ITensor* y = concatLayer->getOutput(0);

    int mip = MAX(8, inp/32);

    /// conv1 *************************************************************************************************************
    auto y_conv = convBlock2(network, weightMap, *y, mip, 1, 1, 1, lname + ".conv1",lname);

    /// torch.split *************************************************************************************************************
    Dims d = y_conv->getOutput(0)->getDimensions();
    ISliceLayer* sliceLayer1 = network->addSlice(*y_conv->getOutput(0), Dims3(0, 0, 0), Dims3(d.d[0], d.d[1] / 2, d.d[2]), Dims3(1, 1, 1));
    ISliceLayer* sliceLayer2 = network->addSlice(*y_conv->getOutput(0), Dims3(0, d.d[1] / 2, 0), Dims3(d.d[0], d.d[1] / 2, d.d[2]), Dims3(1, 1, 1));

    Permutation permutation2 = {0,2,1};
    IShuffleLayer* shuffleLayer2 = network->addShuffle(*sliceLayer2->getOutput(0));
    shuffleLayer2->setFirstTranspose(Permutation(permutation2));
    ITensor* perm2 = shuffleLayer2->getOutput(0);

    /// conv_h conv_w *************************************************************************************************************
    auto a_h = convBlock3(network, weightMap, *sliceLayer1->getOutput(0), inp, 1, 1, 1, lname + ".conv_h");
    auto a_w = convBlock3(network, weightMap, *perm2, inp, 1, 1, 1, lname + ".conv_w");

    ITensor* sum_input_tensors1[] = {&input, a_h->getOutput(0),a_w->getOutput(0)};
    auto ew1 = network->addElementWise(*sum_input_tensors1[0], *sum_input_tensors1[1], ElementWiseOperation::kPROD);
    auto ew2 = network->addElementWise(*ew1->getOutput(0), *sum_input_tensors1[2], ElementWiseOperation::kPROD);

    return ew2;
}

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by zerollzeng** [comment]
1. Could you please try the latest TRT 8.6/9.2?
2. It's hard to debug accuracy from your third-party modified code, could you please try to export the train model to onnx and check it accuracy using polygraphy? e.g. `polygraphy run model.onnx --trt --onnxrt --atol 1e-4 --rtol 1e-4`
